omnipaxos/src/sequence_paxos/follower.rs

```diff
@@ -260,38 +184,17 @@ where
             && self.state.1 == Phase::Accept
             && self.handle_sequence_num(dec.seq_num, dec.n.pid) == MessageStatus::Expected
         {
+            if dec.decided_idx > self.internal_storage.get_accepted_idx() {
+                let new_accepted_idx = self.internal_storage.flush_batch().expect(WRITE_ERROR_MSG);
+                self.handle_flushed_accepted(dec.n, new_accepted_idx);
```

Comment:
same here, check that the flushed was actually >= decided

Reply:
See above comment

Reply:
I liked the code in the above comment so add it here as well.